mscore/mixer.cpp

```diff
@@ -464,47 +477,49 @@ void Mixer::notifyTrackSelected(MixerTrack* track)
       mixerDetails->setTrack(track->mti());
       }
 
-
-//---------------------------------------------------------
-//   writeSettings
-//---------------------------------------------------------
-
-void Mixer::writeSettings()
-      {
-      MuseScore::saveGeometry(this);
-      }
-
-//---------------------------------------------------------
-//   readSettings
-//---------------------------------------------------------
-
-void Mixer::readSettings()
-      {
-      resize(QSize(480, 600)); //ensure default size if no geometry in settings
-      MuseScore::restoreGeometry(this);
-      }
+//
+////---------------------------------------------------------
+////   writeSettings
+////---------------------------------------------------------
+//
+//void Mixer::writeSettings()
+//      {
+//      MuseScore::saveGeometry(this);
+//      }
+//
+////---------------------------------------------------------
+////   readSettings
+////---------------------------------------------------------
+//
+//void Mixer::readSettings()
+//      {
+//      // resize(QSize(480, 600)); //ensure default size if no geometry in settings
+//      MuseScore::restoreGeometry(this);
+//      }
```

Comment:
don't comment out but remove, git will keep the history.

Reply:
or better leave in and write/read the settings?

Reply:
Will remove commented out code. Thought I'd caught all such stragglers on this push :). It's redundant code as even when there's no geometry to read it gets a default from the QT Creator file.

Reply:
@Obliquely would you mind addressing the @Jojo-Schmitz's review?